src/data/minimaps.asm

```diff
@@ -9,18 +9,13 @@
 ; but hopefully a better visual metaphor than raw values.
 
 
-; Fun rgbds gotcha: NEWCHARMAP also switches to it.
-; This is not indicated in the manual.
```

Comment:
It has been [since 0.5.0](https://rgbds.gbdev.io/docs/v0.5.0/rgbasm.5#NEWCHARMAP)! ;)